### ISSUE TITLE
Bug with CIFAR-100 dataset.

### ISSUE THREAD

**Issue by kushagrabhushan** [transmuteAI/trailmet#32]
Trained 3 models with CIFAR-100 created by DatasetFactory but all three gave higher accuracy than expected. Accuracies were close to what we expect from CIFAR-10.

**Comment by sydarb** [comment]
Compared with torchvision cifar dataset, working ok.
[src](https://github.com/heisenbug237/pytorch-cifar100/blob/main/main.py) for ref.
 
![Screenshot 2022-09-11 at 11 27 06 AM](https://user-images.githubusercontent.com/79010410/189514953-73f25ac8-8c2a-4e66-8e75-0a4939345113.png)

**Comment by dkgupta90** [comment]
> Compared with torchvision cifar dataset, working ok. [src](https://github.com/heisenbug237/pytorch-cifar100/blob/main/main.py) for ref.
> 
> ![Screenshot 2022-09-11 at 11 27 06 AM](https://user-images.githubusercontent.com/79010410/189514953-73f25ac8-8c2a-4e66-8e75-0a4939345113.png)

Thanks @heisenbug237 . @gkrm2727 confirmed that the loading of CIFAR10 and CIFAR100 works fine. @kushagrabhushan can you confirm if there is any issue pending with the datasets now? For now, I am putting the issue to resolved.
